meta/documents/changelog_en.md

```diff
@@ -1,5 +1,10 @@
 # Release Notes for Elastic Export Google Shopping
 
+## v1.0.14 (2017-05-19)
+
+### Fixed
+- An issue was fixed which caused the properties not to be in the set language. 
```

Comment:
Sollte das nicht besser lauten "An issue was fixed which caused the properties not to be exported in the configured language."? @hschwab